### ISSUE TITLE
Motion 

### ISSUE THREAD

**Issue by micheletorrini** [Sunoo/homebridge-camera-ffmpeg#920]
motion sensor don't detect someting the doesn't start the switch for notifiers to HomeKit  

**To Reproduce:**

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Config:**

```json
Show your homebridge config.json here.
Remove any sensitive information, such as your homebridge-gsh / google-smarthome token.
```

**Screenshots:**

**Environment:**

* **Node.js Version**: 
* **NPM Version**: 
* **Homebridge Version**: 
* **Homebridge Camera FFmpeg Version**:
* **Homebridge Config UI X Plugin Version**:
* **Operating System**:

**Comment by Sunoo** [comment]
This plugin does not currently have a way of detecting motion on its own, however motion detection can be triggered in various other ways: https://sunoo.github.io/homebridge-camera-ffmpeg/automation/

**Comment by micheletorrini** [comment]
I just Tried but it doesn't work anyway

My json:


{

"cameras": [{

"name": "CamGiardino",

"manufacturer": "SV3C",

"model": "IPCAM",

"firmwareRevision": "V20.1.23.5.3",

"motion": true,

"videoConfig": {

"source": "-re -i rtsp://admin:vs74ya50@192.168.1.183:554/11",

"stillImageSource": "-i rtsp://admin:vs74ya50@192.168.1.183:554/11",

"maxBitrate": 1,

"audio": true

}

},


Inviato da Michele's iPhone

Il giorno 5 dic 2020, alle ore 20:22, David Maher <notifications@github.com> ha scritto:

﻿

Closed #920<https://github.com/Sunoo/homebridge-camera-ffmpeg/issues/920>.
